### ISSUE TITLE
QoL Idea (Menu): C-up idle animation for palette editing

### ISSUE THREAD

**Issue by Bear64DX** [coop-deluxe/sm64coopdx#201]
**TLDR:** Make the character's head stay still in the palette menu.

I think it'd be cool if there was an option you could toggle that changes the idle animation for palette editing. Basically, you could make it so that when you edit a palette, your character always has a straight face (C-up idle, talking to NPCs, etc), or you could make your character move their head (like how it is right now). I think this would be cool because it would allow for cleaner screenshots, where your character would always be facing straight at the screen. It would most likely look something like this, where your character is always facing the screen:
![image](https://github.com/user-attachments/assets/77289f0f-4a6e-4331-a4ba-b1c96337c59b)
I understand that this might not be highest priority due to the menu being buggy, but I would still like to get this out here.

**Comment by AgentXLP** [comment]
![image](https://github.com/user-attachments/assets/af0171d7-8e0b-439b-b15c-ed2c670461c1)
The character already faces the screen in the player menu and its sub menus which makes screenshotting nice, not sure I really get this suggestion.

**Comment by Bear64DX** [comment]
Perhaps I explained that a little poorly. It'd be cool if the character ALWAYS faced the player. The character moves their head side to side, rather than staying still. As seen in this video:

https://github.com/user-attachments/assets/c55fd708-d635-463a-941d-219192ec4b7a

Hope that helps clear things up.
